### PR TITLE
PAASTA-17227 - Adding KeyError fix for SLO Watchers

### PR DESCRIPTION
**Pull request by lanceleih**
### Description
Adding fix to KeyErrors for SLO Watchers threads in Sticht.  Many users have been experiencing KeyErrors whenever they deploy their services. This error occurs whenever we try to call an SLO Watcher with a non existing SLO label.

SLO labels are retrieved through the SignalFx stream. After some investigation, it appears that SignalFx also generates its own SLO label which looks like "_SF_COMP_". As we did not create this SLO label, no SLO watcher with that label actually exists. This results into a KeyError exception being raised.

In other words, the first item in the list of SLO labels that we receive from the SignalFx stream is just arbitrary info that SignalFx generates and therefore can be ignored.

### Tests
`make test` - PASSED

### CODE REVIEW

File: tests/test_slo.py [hunk 1]
```diff
@@ -132,6 +132,17 @@ def test_SLODemultiplexer():
...
+
+    try:
+        demux.process_datapoint(
+            props={'dimensions': {'sf_metric': '_SF_COMP_E123ABC'}}, datapoint=good, timestamp=0.0,
+        )
+    except KeyError:
+        keyErrorRaised = True
+
+    assert keyErrorRaised is False
+
```
Comment (EvanKrall) [inline tests/test_slo.py:145]: You could simplify this to just the `demux.process_datapoint(...)` call - the KeyError would cause the test to fail: https://stackoverflow.com/questions/20274987/how-to-use-pytest-to-check-that-error-is-not-raised

The way you have written this is more explicit, which is good, but if it does start raising a KeyError for some reason, someone's going to have to remove the try/except block anyway, to get a proper traceback so they can debug the failing test.

**Review by EvanKrall — APPROVED** [review]
